File: projects/engineering.purdue.edu/scriptingwithobjects/swocode/chap3/StdErr.py
#!/usr/bin/python

### StdErr.py

import sys                                                          #(A)

while 1:                                                            #(B)
    try:                                                            #(C)
       input = raw_input()                                          #(D)
    except EOFError: break                                          #(E)
    print "You said =>    %s" % input                               #(F)
    length = len( input )                                           #(G)
    sys.stderr.writelines( str(length) + "\n" )                     #(H)

File: projects/demos/engineering.purdue.edu/scriptingwithobjects/swocode/chap2/StdErr.pl
#!/usr/bin/perl -w

### StdErr.pl

use strict;

while ( my $input = <STDIN> ) {                                    #(A)
    print STDOUT "You said =>    $input";                          #(B)
    my $len = length $input;                                       #(C)
    print STDERR "$len\n";                                         #(D)
}
